content/docs/getting-started/installation/operator/modules/authorizationv2-0.md

```diff
@@ -211,38 +205,147 @@ spec:
 
 <br>
 
-3. Install cert-manager CRDs
-    ```bash
-    kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.11.0/cert-manager.crds.yaml
-    ```
+4. Prepare `samples/csm-authorization/config.yaml` which contains the JWT signing secret. The following table lists the configuration parameters.
```

Comment:
Can we add back the location of this samples-config file?

` [samples/authorization/config.yaml](https://github.com/dell/csm-operator/blob/main/samples/authorization/config.yaml)`

Reply:
Fixed